### ISSUE TITLE
Team sync December 11, 2023

### ISSUE THREAD

**Issue by gabalafou** [Quansight-Labs/czi-scientific-python-mgmt#51]
Sync between Smera and Gabriel.

### Notes, decisions made

- Open question: how are we going to make public the PyData Sphinx Theme design system?
  - Gabriel: when I tried opening the URL in another browser, I was taken to a Figma sign in / sign up page: https://www.figma.com/file/BHkBFxg1Qg0h5RApUw1ZrR/PyData-Design-system
- Smera is working on dark mode for tables: https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/50
- Decisions made for header nav icon right side icons:
  - They should all have square focus rings
  - The PyData logo should be like the others
- Decision for hover state on the open buttons for the version switcher and the more dropdown: keep text color the same but brighten/darken it
- It's possible to move focus to the back to top button when tabbing away from the version warning button (Switch to stable version)
  - Gabriel: because of this, I'm going to leave the focus ring that I already added, looks like this: ![back to top button focus ring has the same pill shape and color as the button, with slight gap or offset from the edge of the button](https://github.com/Quansight-Labs/czi-scientific-python-mgmt/assets/317883/8dc1617a-16f5-46ae-93bb-9ddd91ca3480)
- Smera on PTO Wednesday-Friday.

### Action items:

- [x] Gabriel to ask Tania if there is a grayscale version of the PyData logo somewhere
- [x] Smera will check the links in the "More" dropdown to make sure contrast on hover is okay
- [ ] Smera to give some time to think about the hover states for the open buttons for the version switch and the more dropdown
- [x] Black text on orange button: needs further investigation
- [x] Tania to look at the [header nav](https://pydata-sphinx-theme--1545.org.readthedocs.build/en/1545/)[ (#1545) ](https://pydata-sphinx-theme--1545.org.readthedocs.build/en/1545/)[preview build](https://pydata-sphinx-theme--1545.org.readthedocs.build/en/1545/)
- [x] Gabriel to hunt for other buttons on the PST documentation site

**Comment by trallard** [comment]
>  Black text on orange button: needs further investigation

This is a very very low priority - text/background on orange so let's not focus on this @smeragoel

**Comment by trallard** [comment]
21st December 

@gabalafou todo:
 - Collapsible admonitions (toggle button): change chevron (up/down) and add click to collapse text (see Figma)
 - @trallard to review opened PRs and merged
 - Last three items in https://github.com/Quansight-Labs/czi-scientific-python-mgmt/issues/49 will be dealt with separately

@smeragoel 
 - Will work on the design for outline buttons
 - Tables!!!
